index.js

```diff
@@ -18,7 +18,7 @@ try {
 
 function Xvfb(options) {
   options = options || {};
-  this._display = (options.displayNum ? ':' + options.displayNum : null);
+  this._display = (options.displayNum!==undefined ? ':' + options.displayNum : null);
```

Comment:
`!==undefined` is too permissive. It ignores other falsey values such as `""`, `null` and `false`.

If you really want to support `0`, then add `|| options.displayNum === 0`.

If you don't care about `false`, then `options.displayNum != null` would also be fine.

Reply:
I thought this was exactly the function of this line, only check if we do indeed have a value, not validate it. My reasoning was that if anything other than an undefined, int, or string is passed in, something wrong is happening upstream, and I'd rather let it go on and break things than null it silently. It would've helped in my case.
But I can certainly see how that may not always be wise. Some validation is better than no validation, at least, and minimum impact is probably a good idea.